PyStemmusScope/utils.py

```diff
@@ -69,6 +73,94 @@ def to_absolute_path(
     return pathlike.expanduser().resolve(strict=must_exist)
 
 
+def get_forcing_file(config):
+    """Get forcing file from the location.
+    """
+    location, fmt = check_location_fmt(config["Location"])
+    # check if the forcing file exists for the given locaiton(s)
+    if fmt == "site":
+        # get forcing file list
+        forcing_filenames_list = [file.name for file in Path(config["ForcingPath"]).iterdir()]
+        forcing_file = [filename for filename in forcing_filenames_list if location in filename]
+        #pylint: disable=no-else-raise
+        if not forcing_file:
+            raise ValueError(f"Forcing file does not exist for the given site {location}.")
+        elif len(forcing_file) > 1:
+            raise ValueError(f"Multiple forcing files exist for the given site {location}." +
+                "Please check your focing files and remove the redundant files.")
+        else:
+            forcing_file = forcing_file[0]
+
+    elif fmt == "latlon":
+        raise NotImplementedError
+    elif fmt == "bbox":
+        raise NotImplementedError
+
+    return forcing_file
+
+
+def check_location_fmt(loc):
+    """Check the format of location.
+
+    Three types of format are supported:
+    - Site name (e.g. "DE-Kli")
+    - Latitude and longitude (e.g. "(56.4, 112.0)")
+    - Bounding box (lat_min, lat_max), (lon_min, lon_max) (e.g. "[19.5,20.5], [125.5,130.0]")
+
+    Args:
+        loc: String of location extracted from config file.
+
+    Returns:
+        Location name (string) or location (tuple) lat,lon pair and its format
+    """
+    # remove all blanks
+    loc = loc.replace(" ","")
+    if re.fullmatch("[A-Z]{2}-[A-Z][a-z]{2}", loc):
+        location = loc
+        fmt = "site"
+    elif re.fullmatch(r"\(\d*[.,]?\d*,\d*[.,]?\d*\)", loc):
```

Comment:
```suggestion
    elif re.fullmatch(r"/\([+-]?\d*.\d*,\s?[+-]?\d*.\d*\)", loc):
```

Reply:
Thanks for the suggestion. It is revised.